packages/sanes-browser-extension/src/extension/background/model/persona/index.ts

```diff
@@ -84,6 +91,7 @@ export interface MakeAuthorizationCallbacks {
 export interface PersonaAcccount {
   /** human readable address or placeholder text */
   readonly label: string;
+  readonly address: Address;
```

Comment:
One account represents a set of addresses (IOV address, Lisk address, Ethereum adddress), where none of the addresses is the primary one. Do we really need this field?

Reply:
In the design improvements we were supposed to show an address per account, and since we decided to remove the account selector component we thought we should show only one address. I'm not sure which address should be shown and how to get it.

Reply:
okay, cool. In this case let's call the field `iovAddress`